### PR TITLE
phpcs

### PR DESCRIPTION
**Pull request by PhilETaylor**
some more

### CODE REVIEW

File: build/build.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ function usage($command)
 // Parse input options
 $options = getopt('', ['help', 'remote::', 'exclude-zip', 'exclude-gzip', 'exclude-bzip2']);
 
-$remote       = isset($options['remote']) ? $options['remote'] : false;
+$remote       = $options['remote'] ?? false;
```
Comment (richard67) [inline build/build.php]: @PhilETaylor The null coalesce operator is not supported e.g. on PHP 5.3, so we can't do this on staging.

Comment (zero-24) [reply build/build.php]: Yes please revert it here that script is intended to run on php 5.4 right now. I personally have nothing against changing that but that would be out of scope of this PR.

Comment (PhilETaylor) [reply build/build.php]: ah sorry - aint no one got time for staging anymore ;-) I'll revert that bit

Comment (richard67) [reply build/build.php]: Pity we have to support these stone aged PHP versions ... but that's another issue 😄 Personally I love the Elvis and null coalesce operators.

**Review by zero-24 — APPROVED** [review]

**Comment by zero-24** [comment]
Merged thanks

**Pull request merged**